kratos/includes/global_variables.h

```diff
@@ -34,6 +34,7 @@ namespace Kratos
 namespace Globals
 {
 	constexpr double Pi   = 3.14159265358979323846264338327950288419716939937510582097494459230781640628620899862803482534211706798214808651L;
+	constexpr double Epsilon = 1e4 * std::numeric_limits<double>::epsilon();
```

Comment:
I remember when I added this some time ago...

Reply:
I'm not sure how useful this is as a global.

Reply:
Where this expression come form?